jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsExtension.java

```diff
@@ -89,39 +72,18 @@ public void initialize(final ExtensionContext context) {
         final SubsystemRegistration subsystem = context.registerSubsystem(SUBSYSTEM_NAME, CURRENT_MODEL_VERSION);
         final ManagementResourceRegistration registration = subsystem.registerSubsystemModel(JaxrsSubsystemDefinition.INSTANCE);
         registration.registerOperationHandler(GenericSubsystemDescribeHandler.DEFINITION, GenericSubsystemDescribeHandler.INSTANCE);
+        JaxrsContextParamHandler.registerAttributes(registration);
```

Comment:
Since the parser registers this under `<context-parameters>`, I think it makes sense to put all these new attributes under a separate child resource rather than directly in the subsystem root. That way if we ever end up adding a new set of attributes in this subsystem relating to something else, we have a clear separation between the different things. Also, it will make the transformers (which I have not checked yet simpler).

I suggest using PathElement("configuration", "context-parameters") as the element name (@bstansberry I think the plural form is ok here since it contains attributes rather than child resources?)